### ISSUE TITLE
Include server_id in terraform import

### ISSUE THREAD

**Issue by ltickett** [Lucky3028/terraform-provider-discord#74]
When importing the `server_id` isn't included but it is required.

Here's the state after importing a voice channel:

```
terraform import discord_voice_channel.support XXX

# discord_voice_channel.support:
resource "discord_voice_channel" "support" {
    bitrate                  = 64000
    category                 = "ZZZ"
    id                       = "YYY"
    name                     = "Support"
    position                 = 2
    sync_perms_with_category = true
    type                     = "voice"
    user_limit               = 0
}
```

If we then add this to a `.tf` file it fails due to missing `server_id`.

Adding it manually works, but I guess causes an update of sorts?

I would love to contribute a fix, but not sure where to start? I don't know golang but happy to have a go!

Thanks @Lucky3028

**Comment by Lucky3028** [comment]
Ah...I think the `server_id` and `name` are required so that the same `discord_voice_channel` definition can be changed to another `discord_server`, since multiple `discord_servers` can each be defined as a resource. Thus, this behavior seems to be intended.

**Comment by ltickett** [comment]
@Lucky3028 I don't quite follow.

It totally makes sense that the `server_id` is required. What does not make sense, is that the `server_id` isn't imported into the `state` when running `terraform import discord_voice_channel.support XXX`.

**Comment by Lucky3028** [comment]
The `terraform import` records the state of existing resources in `terraform.tfstate`. Thus, `server_id` is stored in `terraform.tfstate`.
However, it does not change the actual definition, so the tf file must contain the required properties for that resource. This is a Terraform specification. https://developer.hashicorp.com/terraform/cli/import#state-only

**Comment by Lucky3028** [comment]
closed by #88
